docs/your_first_api.md

```diff
@@ -715,10 +718,62 @@ An example of simple constants follows - observe our last change to the schema b
 
 ## Be Persistent
 
-So far, we've used an in-memory URI for Datomic. That means just what
-it sounds like: values are only stored in memory. To make it
-persistent, you need to pick a [storage engine](http://docs.datomic.com/storage.html#storage-services) and update the
-`:datomic-uri` value.
+So far, we've used an in-memory URI for Datomic. That means just what it sounds like:
+values are only stored in memory. To make it persistent, you need to pick
+a [storage engine](http://docs.datomic.com/storage.html#storage-services) and update the
+`:datomic-uri` value. For your initial efforts,
+Datomic's [dev storage protocol](http://docs.datomic.com/dev-setup.html) may suffice.
+
+By default, Vase uses the free version of Datomic. In order to configure dev or
+other persistent stores, you will need to first
+[obtain a Datomic Starter or Datomic Pro license](http://www.datomic.com/get-datomic.html)
+and install the software on your machine.
+
+You'll also need to reference datomic-pro in your dependencies. Doing so is somewhat
+non-standard, because the library is not available on [Clojars](https://clojars.org/).
```

Comment:
Datomic provides a Maven repo with password auth, should the instructions use that instead?